File: ClinicSystem/src/com/example/clinicsystem/models/classes/M_Address.java
package com.example.clinicsystem.models.classes;

public class M_Address {
    private int id;
    private String city, streetName, governorate, apartmentNumber;

    public M_Address(int id, String city, String streetName, String governorate, String apartmentNumber) {
        this.id = id;
        this.city = city;
        this.streetName = streetName;
        this.governorate = governorate;
        this.apartmentNumber = apartmentNumber;
    }

    public M_Address() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getStreetName() {
        return streetName;
    }

    public void setStreetName(String streetName) {
        this.streetName = streetName;
    }

    public String getGovernorate() {
        return governorate;
    }

    public void setGovernorate(String governorate) {
        this.governorate = governorate;
    }

    public String getApartmentNumber() {
        return apartmentNumber;
    }

    public void setApartmentNumber(String apartmentNumber) {
        this.apartmentNumber = apartmentNumber;
    }
}

File: clinic/M_Address.php
<?php

class M_Address {
	private $id;
    private $city, $streetName, $governorate, $apartmentNumber;
	
	public function __construct($id, $city, $streetName, $governorate, $apartmentNumber) {
		$this->id=$id;
		$this->city=$city;
		$this->streetName=$streetName;
		$this->governorate=$governorate;
		$this->apartmentNumber=$apartmentNumber;
	}
	
	public function getID() {
        return $this->id;
    }

    public function setID($id) {
        $this->id=$id;
    }

    public function getCity() {
        return $this->city;
    }

    public function setCity($city) {
        $this->city=$city;
    }

    public function getStreetName() {
        return $this->streetName;
    }

    public function setStreetName($streetName) {
        $this->streetName=$streetName;
    }

    public function getGovernorate() {
        return $this->governorate;
    }

    public function setGovernorate($governorate) {
        $this->governorate=$governorate;
    }

    public function getApartmentNumber() {
        return $this->apartmentNumber;
    }

    public function setApartmentNumber($apartmentNumber) {
        $this->apartmentNumber=$apartmentNumber;
    }
}

?>